src/app/member/components/memberActions.ts

```diff
@@ -0,0 +1,21 @@
+"use server";
+import { z } from "zod";
+import { formSchema } from "./MemberEditPage";
+import { getSupabaseClient } from "@/utils/supabase/client";
+import { Tables } from "@/lib/db";
+import { Status } from "@/constant/constant";
+import { revalidatePath } from "next/cache";
+import { Routes } from "@/lib/routes";
+
+export const onSubmit = async (values: z.infer<typeof formSchema>) => {
```

Comment:
Let's call this function and file `createUser`

Reply:
fixed